### DISCUSSION TITLE
v2.0.3 prerelease

### DISCUSSION THREAD

**Discussion by molnard** [molnard/WalletWasabi#49] [General]
[here](https://github.com/molnard/WalletWasabi/releases/tag/v2.0.3rc1)

**Comment by MarnixCroes** [comment]
> 🍏 [Apple M1](https://github.com/zkSNACKs/WalletWasabi/releases/download/v2.0.3/Wasabi-2.0.3-arm64.dmg)

add that it's also for M2
